### PR TITLE
test: Add  tests in Incoming Payment Controller

### PR DESCRIPTION
**Pull request by jocaetano1**
### Context

- closes (or fixes) #682

Test in **IncomingPaymentController**

### Changes

We've added tests to the IncomingPaymentController.

You can check the changes.

@aosccode

### CODE REVIEW

**Comment by trupakufi** [comment]
@rico191013  @raducristianpopa  @Tymmmy  Can you check it please ?

File: packages/wallet/backend/tests/incomingPayment/controller.test.ts [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+      expect(createSpy).toHaveBeenCalledTimes(1)
+      expect(createSpy).toHaveBeenCalledWith(
+        req.session.user.id,
+        req.body.paymentPointerId,
+        req.body.amount,
+        req.body.description,
+        req.body.expiration
+      )
+    })
+    it("should return status 200 and message 'SUCCESS' if the payment was created", async () => {
```
Comment (Tymmmy) [inline packages/wallet/backend/tests/incomingPayment/controller.test.ts:66]: Please add new lines after every test function

Comment (jocaetano1) [reply packages/wallet/backend/tests/incomingPayment/controller.test.ts:66]: Of Course.

Comment (trupakufi) [reply packages/wallet/backend/tests/incomingPayment/controller.test.ts:69]: @Tymmmy  it's allright now ?

File: packages/wallet/backend/tests/rafiki/controller.test.ts [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+      await rafikiController.onWebHook(req, res, (err) => {
+        next()
+        errorHandler(err, req, res, next)
+      })
+
+      expect(next).toBeCalled()
+      expect(next).toBeCalledTimes(1)
+      expect(res.statusCode).toBe(400)
+    })
```
Comment (rico191013) [inline packages/wallet/backend/tests/rafiki/controller.test.ts:137]: can you add new line here?

File: packages/wallet/backend/tests/rafiki/controller.test.ts [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+    it('should not call onWebHook in rafikiService if the request body is invalid', async () => {
+      req.body = mockOnWebhookRequest().body
+      delete req.body.type
+      const onWebHookSpy = jest.spyOn(mockRafikiService, 'onWebHook')
+
+      await rafikiController.onWebHook(req, res, next)
+
+      expect(onWebHookSpy).not.toBeCalled()
+    })
```
Comment (rico191013) [inline packages/wallet/backend/tests/rafiki/controller.test.ts:146]: new line here as well

**Review by rico191013 — APPROVED** [review]

File: packages/wallet/backend/tests/rafiki/controller.test.ts (hunk 2 above)
Comment (trupakufi) [reply packages/wallet/backend/tests/rafiki/controller.test.ts:137]: Yes, of course.

File: packages/wallet/backend/tests/rafiki/controller.test.ts (hunk 3 above)
Comment (trupakufi) [reply packages/wallet/backend/tests/rafiki/controller.test.ts:146]: Perfect.

**Review by trupakufi — COMMENTED** [review]
Making changes.

**Review by trupakufi — APPROVED** [review]

**Comment by trupakufi** [comment]
@Tymmmy @rico191013 changes submited, can you review it again ?

**Comment by trupakufi** [comment]
@Tymmmy @rico191013  changes submited, can you review it again ?

**Review by Tymmmy — DISMISSED** [review]

**Comment by Tymmmy** [comment]
We will merge this after the Interledger Summit. Thank you @jocaetano1 @trupakufi . Follow announcements for all hacktober updates here: https://community.interledger.org/devcer/hacktoberfest-with-interledger-join-the-open-payments-movement-3bkd

**Comment by devcer** [comment]
Thank you for Contributing to Interledger during Hacktoberfest. If you have any accepted PRs, You can claim your Swag [here](https://community.interledger.org/devcer/hacktoberfest-swag-redemption-claim-your-cool-gear-2152). Last date to fill up the form is December 10th, 2023. @jocaetano1

**Pull request closed without merging**